frontend/app/components/videos/ChatMessage.tsx

```
@@ -1,89 +1,118 @@
-                      alt={emoteName}
-                      height={28}
-                    />
-                  </Tooltip>
-                );
-              }
+                  <Text key={`${comment._id}-text-${index}`} className={classes.message} span>
+                    {fragment.text}
+                  </Text>
+                )
+              case GanymedeFormattedMessageType.Emote:
+                const emoteName = fragment.emote?.name || fragment.text;
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)